### PR TITLE
Update reply to

### PR DESCRIPTION
**Pull request by matt72537**

### CODE REVIEW

**Comment by sb8244** [comment]
Howdy from Release Ninja! You should write a release note targeted at a semi-technical end user who is
not familiar with the code base. Include at least 1 image, wherever possible. [Review the help article](https://www.therelease.ninja/help) if
you are not familiar with Release Ninja.

When your pull request is ready, do one of the following:

* [Notify Release Team](https://www.therelease.ninja/workflow/review?project_id=3&pull_request_id=42&repository_id=3)
* Don't notify anyone because it's a small change or doesn't concern them
Whatever you do though, make sure you :tada:

File: app/mailers/notes_mailer.rb [hunk 1]
```diff
@@ -1,4 +1,5 @@
 class NotesMailer < ActionMailer::Base
+  REPLY_TO = ENV.fetch('REPLY_TO_ADDRESS')
   default from: "noreply@therelease.ninja"
```
Comment (matt72537) [inline app/mailers/notes_mailer.rb]: would
```
  REPLY_TO = ENV.fetch('REPLY_TO_ADDRESS')
  default from: REPLY_TO
```
be better here?  Not sure if there are any cases for using noreply@thereleaseninja.com

Comment (sb8244) [reply app/mailers/notes_mailer.rb]: I don't think we want to set the from because it is subject to mail verification then

**Comment by sb8244** [comment]
Howdy from Release Ninja! It appears that you merged without a release note comment? It's never too late!
:tada:

**Comment by sb8244** [comment]
Thanks Matt!

**Pull request merged**